dev/com.ibm.ws.transport.http/src/io/openliberty/http/ext/HttpRequestExt.java

```diff
@@ -0,0 +1,30 @@
+/*******************************************************************************
+ * Copyright (c) 2022 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package io.openliberty.http.ext;
+
+import com.ibm.wsspi.http.HttpRequest;
+import com.ibm.wsspi.http.channel.values.HttpHeaderKeys;
+
+/**
+ *
+ */
+public interface HttpRequestExt extends HttpRequest {
+
+    /**
+     * Access the first instance found for the given header key. This might be
+     * null if no instance was found.
+     *
+     * @param name
+     * @return String
+     */
+    String getHeader(HttpHeaderKeys key);
```

Comment:
```suggestion
    default String getHeader(HttpHeaderKeys key) {
        return getHeader(key.getName());
    }
```

Doing this and similar default impls in HttpResponseExt probably will go a long way in helping with future impls of these interfaces as suggested in my response to Manuel's review comment.